bodhi/server/models.py

```diff
@@ -1259,6 +1272,41 @@ def signed(self):
             return True
         return all([build.signed for build in self.builds])
 
+    @property
+    def ci_status(self):
+        """ Returns a boolean representing if all the builds associated with
+        this update are ignored or have passed their CI testing.
+
+        Returns False if at least one of the tests of the builds associated
+        with this update are queued, running or failed.
+        """
+
+        ci_status = set([build.ci_status for build in self.builds])
+        if not ci_status.difference(set([CiStatus.ignored, CiStatus.passed])):
+            return True
+        return False
```

Comment:
Since the label ```ci_status``` elsewhere would refer to the Enum type, it's a little confusing that this method returns a bool. I suggest renaming the method to ```ci_passed``` so its name sounds like something that should return a bool.